pom.xml

```diff
@@ -516,6 +516,7 @@
                         <version>3.1.2</version>
                         <configuration>
                             <argLine>
+                                @{argLine}
```

Comment:
do we need other arguments somewhere?

Reply:
Jacoco reports stopped getting generated after the version updates of some maven plugins. I don't know if we use them somewhere publicly, but, at least locally they are helpful

Reply:
OK